docker/README.md

```diff
@@ -327,10 +329,40 @@ Instructions on how to build standalone docker images (for development or
 production), outside of `docker-compose`, see the
 [docker docs](../docs/DOCKER.md).
 
+### Using bitcoind version
+If you are using the bitcoind version of the compose file i.e `docker-compose-bitcoind.yml`, follow these additional instructions:
+
+#### Start Bitcoin Node with bitcoind using Docker Compose
+To launch the Bitcoin node using bitcoind in the regtest network using Docker Compose, use the following command:
+```shell
+$  NETWORK="regtest" docker-compose -f docker-compose-bitcoind.yml up
+```
+
+#### Generating RPCAUTH
+Since using `rpcuser` and `rpcpassword` is deprecated in bitcoind core we need to use `rpcauth` instead. You could generate `rpcauth` using [rpcauth.py script](https://github.com/bitcoin/bitcoin/blob/master/share/rpcauth/rpcauth.py) from bitcoin core.
```

Comment:
As mentioned, the `rpcauth` parameter can only be used on the **server** side. On the client side you still need to know username/password.